tensorflow/lite/micro/micro_allocator.cc

```diff
@@ -880,7 +887,7 @@ TfLiteStatus MicroAllocator::CommitStaticMemoryPlan(
 
   const SubGraph* subgraph = model->subgraphs()->Get(subgraph_idx);
   size_t allocation_info_count =
```

Comment:
Do you think it may be worthy to add a comment here that there is a potential optimization here:
ideally, we just need subgraph->tensors()-size() + number of scratch request for the current subgraph.
But we ignore this for this optimization is unlikely to gain much for most realistic mode with large tensors.